migrations/0002-reminders.sql

```diff
@@ -0,0 +1,17 @@
+-- DROP TABLE IF EXISTS reminder_notifications;
+-- DROP TYPE IF EXISTS reminder_types;
+
+CREATE TYPE reminder_types AS ENUM ('recipient_1', 'recipient_2', 'recipient_expired', 'sender_expired', 'sender_1', 'sender_2', 'sender_broadcast_expired', 'recipient_broadcast_expired');
+
+CREATE TABLE IF NOT EXISTS reminder_notifications
+(
+    id              SERIAL PRIMARY KEY,
+    pub_key         CHAR(64) NOT NULL,
+    reminder_type   reminder_types NOT NULL,
+    send_at         timestamp,
```

Comment:
Whole point of the system is to schedule push notifications :P
```suggestion
    send_at         timestamp NOT NULL,
```